src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -94,12 +86,23 @@ pub struct HummockEventHandler {
     shared_buffer_event_receiver: mpsc::UnboundedReceiver<HummockEvent>,
     upload_handle_manager: UploadHandleManager,
     pending_sync_requests: HashMap<HummockEpoch, oneshot::Sender<HummockResult<SyncResult>>>,
+
+    // TODO: replace it with hashmap<id, read_version>
+    _read_version: Option<Arc<RwLock<HummockReadVersion>>>,
 }
 
 impl HummockEventHandler {
     pub fn new(
         local_version_manager: Arc<LocalVersionManager>,
         shared_buffer_event_receiver: mpsc::UnboundedReceiver<HummockEvent>,
+    ) -> Self {
+        Self::new_with_read_version(local_version_manager, shared_buffer_event_receiver, None)
+    }
+
+    pub fn new_with_read_version(
```

Comment:
In our first intermediate stage, there will be only one read version, and the event handle should also hold the read version. So I think there is no case for passing a None read version.